.github/workflows/unit-tests.yml

```diff
@@ -56,9 +61,18 @@ jobs:
 
       - name: install requirements
         run: |
+          set -xe
```

Comment:
Would this step be necessary?

Reply:
I use this when debugging actions.  I can remove it or leave it.